docs/hub/accounts.md

```diff
@@ -1,35 +1,88 @@
 
-#### Mac Installation
+![Run the hub install script.](../images/install/install.png)
 
 On MacOS you will need to confirm that it is okay to run the `hub` binary before it runs successfully. You will see this warning:
 
 ![Select 'Cancel'](../images/install/install_warn.png)
 
-![Go to 'System Settings' => 'General' where you will be able to click, 'Allow Anyway'](../images/install/install_allow.png)
+Select 'Cancel.' 
 
-![Run `hub --help` again and this time select, 'Open` when prompted.](../images/install/install_confirm.png)
+Go to 'System Settings' => 'General' where you will be able to click, 'Allow Anyway.'
+
+![System Settings](../images/install/install_allow.png)
+
+Run `hub --help` again and this time select, 'Open` when prompted.
+
+![Select 'Open` when prompted.](../images/install/install_confirm.png)
 
 You will now be able to continue using the CLI without issue. When you update the binary, you may need to repeat the above steps. We will add developer signing to our MacOS binaries soon.
 
 ![Successfully running `hub --help` to print out the command docs.](../images/hub-cli/hub_help.png)
 
+## Windows Installation
+
+The Windows Hub CLI binary comes with a `hub.exe` that runs `hub` commands. 
+
+You can use [Powershell](https://docs.microsoft.com/en-us/powershell/) to work with the hub CLI by running the executable directly in your terminal like below.
+
+![Running hub --help in Powershell.](../images/install/windows_hub_help.png)
+
+This method requires you to work in the same directory as `hub.exe`.
+
+Another option is to add `hub.exe` to the PATH environment variable to access the `hub` command globally.
+
+### Adding hub.exe to the Path Environment Variable
+
+Place the hub folder in an accessible location like the `C:\` directory.
+
+![Location of hub.exe](../images/install/windows_hub_directory.png)
+
+Open your environment variables option by typing in 'Environment variables' in the Windows start menu and select the first option.
+
+![Search for the environment variables.](../images/install/windows_hub_env1.png)
+
+Select 'Environment Variables' at the bottom.
+
+![Click Environment Variables](../images/install/windows_hub_env2.png)
+
+Select the 'Path' option from User variables if you want to have the hub CLI available only to the current user, or from System variables if you want the hub CLI available for the whole system.
+
+After 'Path' is selected, click 'Edit.'
+
+![Edit your Path.](../images/install/windows_hub_variables.png)
+
+In the 'Edit environment variables' window, select 'New' and enter the location of your hub directory.
+
+![Add hub to environment variables.](../images/install/windows_hub_edit.png)
+
+Click 'OK' to close all the windows.
+
+Open up a new terminal instance and you should be able to run the `hub` command without being in the same directory as the executable.
+
+![Running hub on Powershell.](../images/install/windows_hub_powershell.png)
```

Comment:
very curious now, as I haven't tried on Windows. do the following two commands work for you:

* `hub version`
* `hub update`

?

Reply:
![image](https://user-images.githubusercontent.com/7885328/98745631-ec786a80-2381-11eb-9c00-bc7bef9b8321.png)

![image](https://user-images.githubusercontent.com/7885328/98745663-fa2df000-2381-11eb-8a6c-e9c56a456d8d.png)

Here's what it looks like when I run those.

Reply:
cool, thanks